speech-api/SpeechSynthesis-speak-ownership.html

```diff
@@ -1,10 +1,14 @@
 <!doctype html>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
 <iframe></iframe>
 <script>
 // using an utterance for different SpeechSynthesis instances should throw
-test(t => {
+var t = async_test(
```

Comment:
The previous `test(f(t), name)` structure works for `async_test` as well, just change `test` to `async_test`.

Reply:
Done.